chronos/handles.nim

```diff
@@ -25,6 +25,8 @@ when defined(windows):
     ERROR_PIPE_CONNECTED = 535
     ERROR_PIPE_BUSY = 231
     pipeHeaderName = r"\\.\pipe\chronos\"
+  var IPV6_V6ONLY* {.importc, nodecl, header: "<ws2tcpip.h>".}: cint
```

Comment:
Please take proper constants from `ws2tcpip.h` because this is not what Windows is going to change.

Reply:
Are you suggesting I should copy the value from `ws2tcpip.h`?

Reply:
yes